understanding/non-text-contrast.html

```diff
@@ -1065,17 +856,15 @@ <h4>Infographics</h4>
                   				
                   <section id="symbolic-text-characters">
                      					
-                     <h4>Symbolic text characters</h4>
+                     <h4>記号文字</h4>
```

Comment:
後続で「テキスト文字」としているのであれば、ここも合わせておくべきなのでは
```suggestion
                     <h4>記号テキスト文字</h4>
```